pkg/provisioningv2/rke2/planner/planner_test.go

```diff
@@ -128,3 +128,14 @@ func findEnv(s []string, v string) bool {
 	}
 	return false
 }
+
```

Comment:
Can you make multiple test cases like here, to add both a true and false? https://github.com/rancher/rancher/blob/release/v2.6/pkg/controllers/dashboard/clusterregistrationtoken/agentvars_test.go#L11